pwndbg/gdblib/vmmap.py

```diff
@@ -204,7 +249,18 @@ def explore(address_maybe: int) -> pwndbg.lib.memory.Page | None:
         return None
 
     flags |= 2 if pwndbg.gdblib.memory.poke(address_maybe) else 0
```

Comment:
Fwiw this is kinda stupid, should be:

```suggestion
    if pwndbg.gdblib.memory.poke(address_maybe):
        flags |= 2
```